src/main/java/app/hero/ErrorKeys.java

```diff
@@ -0,0 +1,11 @@
+package app.hero;
+
+public interface ErrorKeys {
+    String EMPTY_NAME = "Name can not be empty.";
+    String STATS_TOO_BIG = "Sum of stats can not be bigger than 10.";
+    String STAT_TOO_BIG = "A statistic can not be bigger than 5.";
+    String STAT_TOO_SMALL = "A statistic can not be a negative value.";
+    int MAX_STAT = 5;
```

Comment:
to nie sa kody bledu a wrzuciles do interfejsu z kodami bledu, po co, dodaj je w swoim validatorze